samples/msal-node-samples/ElectronSystemBrowserTestApp/src/Constants.ts

```diff
@@ -19,6 +19,7 @@ export enum IpcMessages {
     GET_MAIL = "GET_MAIL",
```

Comment:
Since we are not calling /me/messages anymore, can we remove GET/SET_MAIL? (here and elsewhere)

Reply:
Done